server/index.ts

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+import * as next from 'next'
+
+const port: Number = parseInt(<string>process.env.PORT, 10) || 3000
+const dev = process.env.NODE_ENV !== 'production'
+const app = next({ dev })
+const handle = app.getRequestHandler()
+
+app.prepare()
+  .then(() => {
+    const server = express();
```

Comment:
Arguably one of the bigger bummers about using SSR if you want URL parameters: You have to serve the app with a custom Express server, to be able to forward URL parameters to React.

On the bright side, it'll be easy to serve without a special sauce web server.